go.mod

```diff
@@ -4,8 +4,11 @@ go 1.21
 
 require (
 	github.com/creasty/defaults v1.7.0
+	github.com/emersion/go-sasl v0.0.0-20200509203442-7bfe0ed36a21
```

Comment:
No releases? That's a bit unfortunate. But the commit you're referencing isn't the latest one, so you might want to update that one as well.